lib/Round.js

```diff
@@ -29,6 +30,45 @@ class Round {
     });
   }
 
+  getClues(roundCats) {
+    this.clues = roundCats.reduce((acc, categoryId) => {
+      let sortedClues = data.clues.filter(clue => {
+        return clue.categoryId === categoryId;
+      }).sort((a, b) => 0.5 - Math.random());
+
+      this.pointValues.forEach(value => {
+        let chosenClues = sortedClues.find(clue => {
+          return clue.pointValue === value;
+        });
+        acc = acc.concat(chosenClues);
+      });
+      return acc;
+    },[]);
+
+    if (this.roundNum === 3) {
+      this.clues = this.clues[3];
+    }
+  }
+
+  setDailyDouble() {
+    let sortedArray = this.clues.sort((a, b) => 0.5 - Math.random());
+    if (this.roundNum === 2) {
+      sortedArray[0].dailyDouble = true;
+      sortedArray[1].dailyDouble = true;
+    } else {
+      sortedArray[0].dailyDouble = true;
+    }
+  }
+
+  displayClue(categoryId, pointValue) {
+    return this.clues.find(clue => {
+      return (clue.pointValue === pointValue) && (clue.categoryId === categoryId);
+    }).question;
+  }
+
+  validateAnswer(answer, index) {
```

Comment:
I recommend calling to change player scores and changing player turns in the validateAnswer method. If correct: increase and change player; if not correct: decrease score and change player. I think it makes most sense here as it directly affect both those methods. Thoughts?